bindings/generated/constantine_bls12_381.h

```diff
@@ -0,0 +1,80 @@
+/* Generated by Nim Compiler v1.4.6 */
+/*   (c) 2017 Andreas Rumpf */
+/* The generated code is subject to the original license. */
+#ifndef __constantine_bls12_381__
+#define __constantine_bls12_381__
+#define NIM_INTBITS 64
+
+/* section: NIM_merge_HEADERS */
+
+#include "nimbase.h"
+#undef LANGUAGE_C
+#undef MIPSEB
+#undef MIPSEL
+#undef PPC
+#undef R3000
+#undef R4000
+#undef i386
+#undef linux
+#undef mips
+#undef near
+#undef far
+#undef powerpc
+#undef unix
+
+/* section: NIM_merge_FRAME_DEFINES */
+
+/* section: NIM_merge_FORWARD_TYPES */
+typedef struct bls12381_fr bls12381_fr;
+typedef struct bls12381_fp bls12381_fp;
```

Comment:
For proper size in stack the internal fields should be defined. It seems like aliasing breaks that in `type Fr[BLS12_381] = bls12381_fr`

Alternatively can we use exportcpp to properly handle generic exports?

Reply:
> For proper size in stack the internal fields should be defined. It seems like aliasing breaks that in `type Fr[BLS12_381] = bls12381_fr`
> 
> Alternatively can we use exportcpp to properly handle generic exports?

I do think that using export c will simplify the porting of the library in different programming languages. Even if the `exportcpp` is nicer.